src/python/system/shell.py

```diff
@@ -368,6 +368,7 @@ def get_interpreter(file_to_execute, is_blackbox_fuzzer=False):
   # TODO(mbarbella): Remove this when fuzzers have been migrated to Python 3.
   if (is_blackbox_fuzzer and interpreter == sys.executable and
       environment.get_value('USE_PYTHON2_FOR_BLACKBOX_FUZZERS') and
+      not environment.get_value('USE_PYTHON3_FOR_FUZZER') and
```

Comment:
We should remove USE_PYTHON2_FOR_BLACKBOX_FUZZERS from global and then add in environment attribute of all fuzzers
.environment += '\nUSE_PYTHON2_FOR_BLACKBOX_FUZZERS = True'
that way, nobody using clusterfuzz does not have to set any of these env vars and are on python 3 by default.

Reply:
If we plan to set them per-fuzzer I'd rather just have an override for the interpreter. I'm still not totally sure this is worth it though, since the default is python 3 and other users won't run into this at all. Might be easier to discuss in person, but I'm fine with either option.